src/requests/requests.js

```diff
@@ -30,3 +31,19 @@ export const fetchMovies = async (searchQuery, selectedSortOption, selectedGenre
     }
   }
 };
+
+export const fetchMovie = async ({ params }) => {
+  const response = await axios.get(`http://localhost:4000/movies/${params.movieId}`);
+
+  const mappedMovie = {
+    movieName: response.data.title,
+    rating: response.data.vote_average,
+    imageUrl: response.data.poster_path,
+    releaseYear: response.data.release_date,
+    duration: response.data.runtime,
+    description: response.data.overview,
+    genres: response.data.genres,
+  };
```

Comment:
It probably makes sense to create a mapper, that can be used in both `fetchMovies` and `fetchMovie`.